openfl-tutorials/Federated_Keras_MNIST_Tutorial.ipynb

```diff
@@ -10,11 +10,16 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "#Install dependencies if not already installed\n",
-    "!pip install tensorflow mnist"
+    "#Install Tensorflow and MNIST dataset if not installed\n",
+    "!pip install tensorflow==2.3.1 mnist\n",
+    "\n",
+    "#Alternatively you could use the intel-tensorflow build\n",
+    "# !pip install intel-tensorflow==2.3.1 mnist"
```

Comment:
If I would run it I get the following:
```
ERROR: Could not find a version that satisfies the requirement intel-tensorflow==2.3.1
ERROR: No matching distribution found for intel-tensorflow==2.3.1
```

Reply:
Ack! should be intel-tensorflow==2.3.**0**, I will also try out 2.4.0 real quick to see if there are any issues with the latest build and update accordingly, thanks!

Reply:
2.4.0 works, but does throw several dependency errors in the console about numpy and tensorflow-estimator version conflicts. It's probably best for the end user experience to keep intel-tensorflow at 2.3.0 since this tutorial doesn't depend on anything from the latest build.

Reply:
Thanks for the contribution @kshannon. Yes, please keep intel-tensorflow 2.3.0 for now. We currently rely on a gRPC version that has conflicts with 2.4.0 (this should be resolved soon).